src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -444,6 +445,13 @@ namespace HVACCooledBeam {
                                 "Average",
                                 CoolBeam(CBNum).Name);
 
+            SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
+                                OutputProcessor::Unit::m3_s,
+                                CoolBeam(CBNum).OutdoorAirFlowRate,
+                                "System",
+                                "Average",
+                                CoolBeam(CBNum).Name);
+
```

Comment:
Added a new variable to single duct air terminal *AirTerminal:SingleDuct:ConstantVolume:CooledBeam*.